v0/destinations/ga/transform.js

```diff
@@ -48,25 +59,84 @@ function responseBuilderSimple(
   const rawPayload = {
     v: "1",
     t: hitType,
-    tid: destination.Config.trackingID
+    tid: destination.Config.trackingID,
+    ds: message.channel,
+    an: message.context.app.name,
+    av: message.context.app.version,
+    aiid: message.context.app.namespace
   };
+  if (destination.Config.doubleClick) {
+    rawPayload.npa = 1;
+  }
+  if (destination.Config.anonymizeIp) {
+    rawPayload.aip = 1;
+  }
+  if (destination.Config.enhancedLinkAttribution) {
+    rawPayload.linkid = "linkid.js"; // not sure
```

Comment:
instead of the constant value, take it from the `properties`

Reply:
done